### PR TITLE
Update Russian translation

### PR DESCRIPTION
**Pull request by vantu5z**

### CODE REVIEW

File: appIcons.js [hunk 1]
```diff
@@ -1856,27 +1856,27 @@ var MyShowAppsIconMenu = Utils.defineClass({
...
 
             _appendItem(this, {
-                title: 'Device Management',
+                title: _('Device Management'),
                 cmd: ['gnome-control-center', 'display']
             });
 
             _appendItem(this, {
-                title: 'Disk Management',
+                title: ('Disk Management'),
```
Comment (eshagh79) [inline appIcons.js:1879]: Did you forget the _ character?

**Comment by vantu5z** [comment]
@eshagh79 thanks, you are right I forget this line.

File: appIcons.js (hunk 1 above)
Comment (vantu5z) [reply appIcons.js]: fixed by 7d26d0ae233ee54c3a8436c6e46974bbce465899

**Comment by charlesg99** [comment]
I'd like to merge this, could you fix the conflict?

**Comment by vantu5z** [comment]
Fix conflict and update translation.

**Comment by charlesg99** [comment]
Thanks!

**Pull request merged**